platform-emf/src/main/java/com/softicar/platform/emf/form/IEmfForm.java

```diff
@@ -80,4 +80,14 @@
 	 * @return an {@link IEmfValidator} (never <i>null</i>)
 	 */
 	Collection<IEmfValidator<R>> getAdditionalValidators();
+
+	default void enterEditMode() {
+
+		//nothing per default
+	}
+
+	default void enterViewMode() {
+
+		//nothing per default
+	}
 }
```

Comment:
We should aim for the `IEmfForm` interface to be as basic as possible, I suggest to replace the `getFrame()` method with a `close` or `closeForm` method.